index.js

```diff
@@ -1,20 +1,440 @@
+  ["move", "use", "open", "describe"],
+  "Sorry, this door doesn’t open like other doors--just by a turn of a handle or a press of a button--The moss will help you if you ask",
+  "door",
+  ["bird", "moss dome"]
+);
+
+let mossDome = new Room(
+  "moss dome",
+  "A moss dome. You’ve heard of this place, a vast anti-kingdom, nestled within the crook of every Bird’s wing. You want so badly to go inside. You notice a door...",
+  ["move", "open", "use", "describe"],
+  "Sorry, this door doesn’t open like other doors--just by a turn of a handle or a press of a button--The moss will help you if you ask",
+  ["door", "moss", "teacup", "fountain", "mirror"],
+  ["path", "crawlspace"]
+);
+
+let crawlspace = new Room(
+  "crawlspace",
+  "dingy and damp. Just big enough for you to crawl through.",
+  ["move", "enter", "use", "describe"],
+  "The crawlspace brings you into a Large Hall. You can’t help but shout “ECHO!” And a melodic “ECHO ECHO ECHO” returns your call. It’s dark in here and you trip over an unknown object.",
+  [" "],
+  ["moss dome", "great hall"]
+);
+
+let greatHall = new Room(
+  "great hall",
+  `A large Hall. You can’t help but shout "ECHO!” And a melodic “ECHO ECHO ECHO” returns your call.\n
+  It’s dark in here and you trip over an unknown object.`,
+  ["move", "use", "take", "describe"],
+  ["flashlight", "painting"],
+  ["crawlspace", "kitchen"]
+);
+
+let kitchen = new Room(
+  "kitchen",
+  `When you climb through the window, you find yourself in a cozy kitchen with four doors--one on each wall.\n
+  Each door a plaque with its respective cardinal direction--north, south, east, west.\n
+  The smells of your favorite foods fill the small room. The table is set for one.\n
+  Your name is handwritten on a nametag in front of the plate and beckons you to sit.\n
+  You are famished.`,
+  ["move", "use", "eat", "describe"],
+  [" "],
+  ["plate", "nametag", "north door", "south door", "east door", "west door"],
+  ["great hall", "north room", "south room", "east room", "west room"]
+
+);
+
+/*------------------------------------Create FOUR other ROOMS ---------------------------------------------------*/
+
+let northRoom = new Room();
+
+let southRoom = new Room();
+
+let eastRoom = new Room();
+
+let westRoom = new Room();
+
+/*------------------------------------Create ITEMs (passing all properties down from ITEM CLASS ) ---------------------------------------------------*/
+// name of item,
+// description,
+// action (verbs allowed to be used on the item)
+// actionOutcome (response prompt after action is complete),
+// takeable (can the item be taken into player inventory)
+
+/*------------------------------------Create Player Object ---------------------------------------------------*/
+// The object -- playerObject -- has two KEY-VALUE pairs: currentRoom is Key and "forest" is value. playerInventory is key and empty array is value.
+let playerObject = {
+  currentRoom: "forest",
+  playerInventory: [],
```

Comment:
Love this 'help' functionality! I might have to steal this for my own adventure game.